crates/uv-static/src/env_vars.rs

```diff
@@ -615,6 +615,27 @@ impl EnvVars {
     /// Used for trusted publishing via `uv publish`. Contains the oidc request token.
     pub const ACTIONS_ID_TOKEN_REQUEST_TOKEN: &'static str = "ACTIONS_ID_TOKEN_REQUEST_TOKEN";
 
+    /// Indicates that the current process is running in GitLab CI.
+    ///
+    /// When set (typically to `true`), uv may attempt GitLab-specific trusted publishing flows.
+    pub const GITLAB_CI: &'static str = "GITLAB_CI";
+
+    /// GitLab CI/CD-provided JWT for the current job (v2 format).
+    ///
+    /// When using GitLab's `id_tokens` feature with audience set for PyPI, this token can be
+    /// submitted to PyPI's `/_/oidc/mint-token` endpoint to obtain a short-lived publish token.
+    pub const CI_JOB_JWT_V2: &'static str = "CI_JOB_JWT_V2";
+
+    /// GitLab CI/CD-provided JSON Web Token (JWT) for the current job (legacy format).
+    pub const CI_JOB_JWT: &'static str = "CI_JOB_JWT";
```

Comment:
I don't believe this will work, unfortunately -- both `CI_JOB_JWT` and `CI_JOB_JWT_V2` are deprecated, and neither will work with Trusted Publishing on PyPI because they lack the appropriate `aud` claim (i.e. `pypi`).

Instead of using these, uv needs to check for `PYPI_ID_TOKEN`, which the user has to explicitly declare in their GitLab job definition under `id_tokens`.

There's an example of that here: https://docs.pypi.org/trusted-publishers/using-a-publisher/#gitlab-cicd

More details here: https://docs.gitlab.com/ci/yaml/#id_tokens